src/test/java/lotto/model/LottoFactoryTest.java

```diff
@@ -18,6 +21,18 @@ void setup() {
     @Test
     @DisplayName("입력수량만큼 로또번호 리스트를 생성한다")
     void lottoList() {
-        assertThat(generator.addLotto(10).size()).isEqualTo(10);
+        Lotto buyLotto1 = toLottoNumber(List.of(3, 5, 10, 12, 16, 33));
+        Lotto buyLotto2 = toLottoNumber(List.of(4, 6, 10, 11, 13, 18));
+        Lotto buyLotto3 = toLottoNumber(List.of(7, 8, 14, 18, 19, 35));
+
+        assertThat(generator.addLotto(10, getManualLotto(buyLotto1, buyLotto2, buyLotto3)).size()).isEqualTo(13);
+    }
+
+    private List<Lotto> getManualLotto(Lotto buyLotto1, Lotto buyLotto2, Lotto buyLotto3) {
+        return List.of(buyLotto1, buyLotto2, buyLotto3);
+    }
+
+    private Lotto toLottoNumber(List<Integer> testLotto) {
+        return new Lotto(toLottoNumberSet(testLotto));
     }
```

Comment:
이 부분은 stream으로도 충분하지 않을까 싶어요. 
```suggestion
    private Lotto toLottoNumber(List<Integer> testLotto) {
        return new Lotto(
            testLotto.stream()
                .map(LottoNumber::cacheLottoNumber)
                .collect(Collectors.toSet())
        );
    }
```